src/Parser/Monk/Windwalker/Modules/Items/TheEmperorsCapacitor.js

```diff
@@ -65,6 +68,21 @@ class TheEmperorsCapacitor extends Analyzer {
     }
   }
 
+  _isPermanentPet(guid) {
+      return Object.keys(PETS).map(key => PETS[key]).every(pet => pet.id !== guid);
```

Comment:
it looks like the only pets listed in the PETS object are Warlock and Priest things, so won't this always return true for a Monk's pet? Also why is this function called `_isPermanentPet`? I really don't get what this is for.